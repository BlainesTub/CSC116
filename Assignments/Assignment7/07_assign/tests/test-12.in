cos(x)
0.5