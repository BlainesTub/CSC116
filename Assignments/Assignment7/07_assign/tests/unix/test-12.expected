f(x) = cos(x)
f(0.5) = 0.877583
f'(x) = -1*sin(x)*1
f'(0.5) = -0.479426
